src/libs/deprecatedAPI.js

```diff
@@ -671,13 +671,6 @@ function GetCurrencyList() {
     return Mobile_GetConstants({data: ['currencyList']});
 }
 
-/**
- * @returns {Promise}
- */
-function User_IsUsingExpensifyCard() {
```

Comment:
We are still calling `User_IsUsingExpensifyCard` in [validateBankAccount.js](https://github.com/Expensify/App/blob/d2c03dc7224d5c8afa7baf9ec4308b805373b831/src/libs/actions/ReimbursementAccount/validateBankAccount.js#L20), so we can't remove this just yet. We have to leave this for part 2!

Reply:
Good catch, just fixed it. You can review it now. Thanks